recipes-browser/packagegroups/packagegroup-wpewebkit-depends.bb

```diff
@@ -0,0 +1,281 @@
+    sqlite3 \
+    zlib \
+    libpng \
+    libsoup-2.4 \
+    libwebp \
+    libxml2 \
+    libxslt \
+    libepoxy \
+    libgcrypt \
+    \
+    bubblewrap \
+    xdg-dbus-proxy \
+    libseccomp \
+    libicudata \
+"
+
+RDEPENDS_packagegroup-wpewebkit-depends-desktop = "\
+    libxt \
+    libxxf86vm \
+    libdrm \
+    libglu \
+    libxi \
+    libxtst \
+    libx11-locale \
+    xorg-minimal-fonts \
+    gdk-pixbuf-loader-ico \
+    gdk-pixbuf-loader-bmp \
+    gdk-pixbuf-loader-ani \
+    gdk-pixbuf-xlib \
+    liberation-fonts \
+    atk \
+    at-spi2-atk \
+    alsa-lib \
+"
+
+RDEPENDS_packagegroup-wpewebkit-depends-runtime-add = "\
+    ldd \
+    e2fsprogs-mke2fs \
+    mkfontdir \
+    liburi-perl \
+    libxml-parser-perl \
+    libxml-perl \
+    libxml-sax-perl \
+    glibc-localedatas \
+    glibc-gconvs \
+    glibc-charmaps \
+    glibc-binaries \
+    glibc-localedata-posix \
+    glibc-extra-nss \
+    glibc-pcprofile \
+"
+
+RDEPENDS_packagegroup-wpewebkit-depends-alternative = " \
+    geoclue \
+    libtasn1 \
+    woff2 \
+    libvpx \
+    libevent \
+    libopus \
+    openjpeg \
+"
+
+RDEPENDS_packagegroup-wpewebkit-depends-video = " \
+    gstreamer1.0 \
+    gstreamer1.0-plugins-base \
+    gstreamer1.0-plugins-good \
+    gstreamer1.0-plugins-bad \
+    gstreamer1.0-plugins-good-isomp4 \
+    gstreamer1.0-plugins-good-wavparse \
+    gstreamer1.0-plugins-base-opengl \
+    gstreamer1.0-plugins-base-app \
+    gstreamer1.0-plugins-base-playback \
+    gstreamer1.0-plugins-good-souphttpsrc \
+    gstreamer1.0-plugins-base-audioconvert \
+    gstreamer1.0-plugins-base-audioresample \
+    gstreamer1.0-plugins-base-gio \
+    gstreamer1.0-plugins-base-videoconvert \
+    gstreamer1.0-plugins-base-videoscale \
+    gstreamer1.0-plugins-base-volume \
+    gstreamer1.0-plugins-base-typefindfunctions \
+    gstreamer1.0-plugins-good-audiofx \
+    gstreamer1.0-plugins-good-audioparsers \
+    gstreamer1.0-plugins-good-autodetect \
+    gstreamer1.0-plugins-good-avi \
+    gstreamer1.0-plugins-good-deinterlace \
+    gstreamer1.0-plugins-good-interleave \
+    gstreamer1.0-plugins-bad-dashdemux \
+    gstreamer1.0-plugins-bad-mpegtsdemux \
+    gstreamer1.0-plugins-bad-smoothstreaming \
+    gstreamer1.0-plugins-bad-videoparsersbad \
+"
+
+RDEPENDS_packagegroup-wpewebkit-depends-extra = " \
+    ca-certificates \
+    shared-mime-info \
+    ttf-bitstream-vera \
+    gstreamer1.0-plugins-base-meta \
+    gstreamer1.0-plugins-good-meta \
+    gstreamer1.0-plugins-bad-meta \
+"
+
+RDEPENDS_packagegroup-wpewebkit-depends-tests = " \
+    pkgconfig \
+    p7zip \
+    git \
+    subversion \
+    bzip2 \
+    apache2 \
+    apache2-scripts \
+    curl \
+    gdb \
+    php \
+    python \
+    python-subprocess32 \
+    python-pygobject \
+    python-psutil \
+    psmisc \
+    pulseaudio \
+    pulseaudio-misc \
+    highline \
+    perl \
+    perl-module-file-spec \
+    perl-module-cgi \
+    json \
+    dicts \
```

Comment:
I think this name "dicts" is too much generic. I would instead use something like "test-dicts"

Reply:
done